src/Transport/Topology/Topologies/ForwardingTopologySectionManager.cs

```diff
@@ -16,12 +17,21 @@ class ForwardingTopologySectionManager : ITopologySectionManager
         readonly ConcurrentDictionary<string, TopologySection> sendDestinations = new ConcurrentDictionary<string, TopologySection>();
         readonly ConcurrentDictionary<Type, TopologySection> publishDestinations = new ConcurrentDictionary<Type, TopologySection>();
         readonly List<EntityInfo> topics = new List<EntityInfo>();
-        readonly Random randomGenerator = new Random();
+        Lazy<NamespaceBundleConfigurations> namespaceBundleConfigurations;
 
         public ForwardingTopologySectionManager(SettingsHolder settings, ITransportPartsContainer container)
         {
             this.settings = settings;
             this.container = container;
+
+            namespaceBundleConfigurations = new Lazy<NamespaceBundleConfigurations>(() =>
+            {
+                var manageNamespaceManagerLifeCycle = container.Resolve<IManageNamespaceManagerLifeCycle>();
+                var namespaceConfigurations = settings.Get<NamespaceConfigurations>(WellKnownConfigurationKeys.Topology.Addressing.Namespaces);
+                var bundlePrefix = settings.Get<string>(WellKnownConfigurationKeys.Topology.Bundling.BundlePrefix);
+                var bundleConfigurations = NumberOfTopicsInBundleCheck.Run(manageNamespaceManagerLifeCycle, namespaceConfigurations, bundlePrefix).GetAwaiter().GetResult();
```

Comment:
Discussed this with @SeanFeldman and we said that on develop the section manager has to become async and this code probably will use `AsyncLazy` as well as not use the container